### PR TITLE
LPS-135819 Drag & drop for files in documents and media does not work with this enabled

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
the Drag and drop fix introduced a bug on the review / expiration Date causing always be "never review/expirate"

### CODE REVIEW

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by Hanlf** [comment]
ci:test:relevant

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/EditFileEntryMVCActionCommand.java [hunk 1]
```diff
@@ -865,7 +865,8 @@ private HttpServletRequest _getDDMStructureHttpServletRequest(
 	}
 
 	private Date _getExpirationDate(
-			UploadPortletRequest uploadPortletRequest, TimeZone timeZone)
+			UploadPortletRequest uploadPortletRequest, boolean addDynamic,
```
Comment (adolfopa) [inline modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/EditFileEntryMVCActionCommand.java:868]: Rename `addDynamic` to something that represents better its function, like "neverExpireDefaultValue" or even "neverExpire".

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/EditFileEntryMVCActionCommand.java [hunk 2]
```diff
@@ -1205,9 +1207,11 @@ private FileEntry _updateFileEntry(
 			User user = _userLocalService.getUser(themeDisplay.getUserId());
 
 			Date expirationDate = _getExpirationDate(
-				uploadPortletRequest, user.getTimeZone());
+				uploadPortletRequest, cmd.equals(Constants.ADD_DYNAMIC),
```
Comment (adolfopa) [inline modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/EditFileEntryMVCActionCommand.java:1210]: Conditional expressions like `cmd.equals(...)` are usually avoided; you need to use a conditional statement:
```
boolean addDynamic = false;

if (cmd.equals(...)) {
   addDynamic = true;
}
```
and then use the variable.

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/EditFileEntryMVCActionCommand.java (hunk 2 above)
Comment (AliciaGarciaGarcia) [reply modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/EditFileEntryMVCActionCommand.java]: ok

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/EditFileEntryMVCActionCommand.java (hunk 1 above)
Comment (AliciaGarciaGarcia) [reply modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/EditFileEntryMVCActionCommand.java]: I'm the very best naming, I know :D

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Pull request closed without merging**